firedrake/slate/slac/compiler.py

```diff
@@ -483,15 +485,15 @@ def tensor_assembly_calls(builder):
         # The mesh level is an integer provided as a macro kernel
         # argument.
 
-        # FIXME: No variable layers assumption
```

Comment:
This FIXME still applies (variable layers is different numbers of layers for each entry in the iteration set).